security/signature_validation/signature_validation.1.x.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/twilio/twilio-go/client"
+)
+
+func main() {
+	// You can find your Auth Token at twilio.com/console
+	// For this example: authToken := "12345"
+	authToken := os.Getenv("TWILIO_AUTH_TOKEN")
+
+	requestValidator := client.NewRequestValidator(authToken)
+
+	// Twilio's request URL
+	url := "https://mycompany.com/myapp.php?foo=1&bar=2"
+
+	// Post variables in Twilio's request
```

Comment:
Is "post" a verb here? I'm not sure what this comment means

Reply:
Maybe we can ask this guy? 
<img width="1115" alt="image" src="https://user-images.githubusercontent.com/10067791/229149081-85aaf1ee-08cb-402b-a391-2679cc1687e2.png">

Reply:
lol how about "save the data from Twilio's request in a variable" or something?

Reply:
Oh my... when would Twilio ever even send query params in the url itself? 😬 
<img width="456" alt="image" src="https://user-images.githubusercontent.com/10067791/229154357-fd3ae61f-2939-4f24-80a0-de56e4526c80.png">